src/org/thoughtcrime/securesms/PassphraseRequiredActionBarActivity.java

```diff
@@ -38,4 +44,5 @@ protected final void onCreate(Bundle savedInstanceState) {
   }
 
   protected void onCreate(Bundle savedInstanceState, boolean ready) {}
+  protected boolean allowInLockedMode() { return false; }
```

Comment:
this requires a comment, what the "locked mode" is etc. and why things are not allowed by default and when one wants to change default behaviour

Reply:
added a comment, to be honest I am not 100% sure of how this "lock mode" hack work so it is hard to explain it XD